projects/ngx-drag-to-select/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tslint.json",
+  "rules": {
+    "directive-selector": [false, "attribute", "ngx", "camelCase"],
```

Comment:
This rule is disabled for now. 
When we rename `selectItem` to something more suitable, we'll have to turn this back on.